README.md

```diff
@@ -27,54 +30,102 @@ Compatibility
 | 2.8.x and greater  | 1.0.2          |
 | 2.7.x              | 0.6            |
 
-The plugins might be compatible with earlier Nexus versions, but are not tested.
+The plugins might be compatible with earlier Nexus versions, but are
+not tested.
 
 
 Debian Packages from a Maven Build Process
 ------------------------------------------
-https://github.com/sannies/blogger-java-deb is a small example on how to create debs in a 
-Maven process and works together well with this plugin.
+https://github.com/sannies/blogger-java-deb is a small example on how
+to create debs in a Maven process and works together well with this
+plugin.
 
 
 Pitfall
 -------
 
-The indexer cannot find packages when there is a main artifact with the same name:
+The indexer cannot find packages when there is a main artifact with
+the same name:
+
 If the artifacts are named like:
 
 -  nexus-apt-plugin-0.5.jar
--  nexus-apt-plugin-0.5.deb
+-  nexus-apt-plugin_0.5.deb
 
-The indexer won't index the debian package. In order to make the indexer index the debian 
-package it needs a classifier:
+The indexer won't index the debian package. In order to make the
+indexer index the debian package it needs a name, a version and an
+architecture, separated by underscores.
 
 -  nexus-apt-plugin-0.5.jar
--  nexus-apt-plugin-0.5-all.deb
+-  nexus-apt-plugin_0.5_all.deb
 
 This is fine.
 
 Known Bug
 ---------
 
-When maven uploads an artifact to Nexus, it uploads 2 files: the deb file
-and the pom. Nexus then indexes the artifact, but it does this for the 2 files simultaneously.
-Depending on which file is uploaded last (seems to be random), the debian package information
-may or may not be in the index and therefore may or may not be in the Packages/Packages.gz
-files.
-
-To fix the index contents, start an 'Update Index' action on the repository.
-
-This is an issue with the indexer in Nexus 2.x.y. Sonatype is planning to use a new indexer
-technology in their Nexus 3.0 release.
+When maven uploads an artifact to Nexus, it uploads 2 files: the deb
+file and the pom. Nexus then indexes the artifact, but it does this
+for the 2 files simultaneously.  Depending on which file is uploaded
+last (seems to be random), the debian package information may or may
+not be in the index and therefore may or may not be in the
+Packages/Packages.gz files.
+
+TODO:
```

Comment:
We should not have TODOs in the README file. You created a TODO file in another commit and there are issue trackers (github) for stuff like this.